README.md

```
@@ -330,6 +331,15 @@ We welcome contributions! Please see our [Contributing Guide](CONTRIBUTING.md) f
 
 For development setup and commands (lint, test, build), see the [Development Guide](docs/guide/getting-started/development.md).
 
+## Contributors
+
+Thanks to all the people who have contributed to Clouisle:
+
+<a href="https://github.com/clouisle/Clouisle/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=clouisle/Clouisle" />
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)